src/GitHubCodeReviewDashboard/GitHubCodeReviewDashboard/wwwroot/js/site.js

```diff
@@ -0,0 +1,4 @@
+﻿// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
+// for details on configuring this project to bundle and minify static web assets.
+
+// Write your Javascript code.
```

Comment:
don't tell me what to do.

Reply:
I left this file in wondering if somebody would say that!